packages/config/config/devices/0x0258/nas-pd02z-2.json

```diff
@@ -0,0 +1,261 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NAS_PD02Z-2",
+	"description": "PIR Motion, Temperature & Light Sensor",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0x1036"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Group 1 is lifeline group; all nodes which associated in this group will receive the messages sent by device through lifeline.",
```

Comment:
We don't actually use the description. Should probably remove it from the import script.
```suggestion
```